src/App.js

```diff
@@ -30,39 +28,9 @@ const SplashOrRouter = memo(({ store }) => {
 })
 
 const App = () => {
+  useServiceWorker()
+  log.debug({ Config })
   const store = SimpleStore.useStore()
```

Comment:
```suggestion
  const store = SimpleStore.useStore()
  
  useServiceWorker()
  useEffect(() => log.debug({ Config }), [])
```

1. if you're using some values from the hook - include those hooks first
2. do not do anything (event the logging) in the component's body. You could just perform some simple calculations / processings to prepare some vars to be passed to the jsx. And **nothing more**. Other stuff should be wrapped onto functions / effects. For logging you could do useEffect(() => log.xxx('msg', data), []) (to log once on mount) or  useEffect(() => log.xxx('msg', data)) (to log after each update)